### PR TITLE
Pos addons 12.0 pos chat durak new

### PR DESCRIPTION
**Pull request by Enigma228322**
fixin bugs

### CODE REVIEW

File: pos_durak/static/src/js/durak_button.js [hunk 1]
```diff
@@ -627,18 +687,26 @@ function SetPos(avatar, uid){
             'margin-bottom: 0px;">'+message+'</p>';
         out += audio_mes;
         out += '</div>';
-        messages.innerHTML += out;
-        setTimeout(delete_message,15000, chat_users[i].msg_cnt, uid);
+        try{
+            messages.innerHTML += out;
+        }
```
[the remark this answers is not in the document]
Comment (Enigma228322) [reply pos_durak/static/src/js/durak_button.js]: It ain't much, bit its honest work :man_farmer:

File: pos_durak/static/src/js/durak_button.js [hunk 2]
```diff
@@ -369,10 +424,14 @@ function ShowUsers(){
         i = NumInQueue(item.uid);
         var str_uid = String(item.uid);
         out += '<div class="chat-user" id="picture-'+i+'">';
-        out += '<div class="user-name" id="user-name-'+str_uid+'">'+chat_users[i].name+'</div>';
+        if(item.ready){
+            out += '<div class="user-name" id="user-name-'+str_uid+'" style="background:green" >'+chat_users[i].name+'</div>';
+        }else{
+            out += '<div class="user-name" id="user-name-'+str_uid+'">'+chat_users[i].name+'</div>';
```
[the remark this answers is not in the document]
Comment (Enigma228322) [reply pos_durak/static/src/js/durak_button.js]: SharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharikSharik

[the remark this answers is not in the document]
Comment (Enigma228322) [reply pos_durak/static/src/js/durak_button.js]: i hate Sharik

File: pos_durak/__manifest__.py [hunk 3]
```diff
@@ -15,13 +15,14 @@
     "license": "LGPL-3",
 
     # any module necessary for this one to work correctly
-    'depends': ['point_of_sale', 'pos_longpolling', 'web_tour'],
+    'depends': ['point_of_sale', 'pos_longpolling'],
```
Comment (yelizariev) [inline pos_durak/__manifest__.py]: wrong

**Pull request merged**